6/changelog.yaml

```
@@ -86,3 +86,9 @@ releases:
 
         `Porting Guide <https://docs.ansible.com/ansible/devel/porting_guides.html>`_'
     release_date: '2022-08-02'
+  6.3.0:
+    changes:
+      deprecated_features:
+      - The google.cloud collection is considered unmaintained and will be
+        removed from Ansible 8 if nobody starts maintaining it again until then
```

I guess it's best to also link to the process for details:
```suggestion
        removed from Ansible 8 if nobody starts maintaining it again until then.
        See `the removal process for details on how this works
        <https://github.com/ansible-collections/overview/blob/main/removal_from_ansible.rst#cancelling-removal-of-an-unmaintained-collection>`__
```